### ISSUE TITLE
Правильное название классов

### ISSUE THREAD

**Issue by Roman52** [bem/bem-forum-content-ru#947]
Здравствуйте!
Изучаю бэм, возник вопрос - правильно ли делать так:

<div class="info-box">
	<div class="info-box__item info-box__item_time">Время</div>
	<div class="info-box__item info-box__item_money">Деньги</div>
	<div class="info-box__item info-box__item_number">Номер</div>
</div>

Смущает _time; _money; _number в названии классов.
Так можно делать?
Естественно при условии, что на _time; _money; _number навешиваются доп стили, на каждый элемент свой стиль.

**Comment by vithar** [comment]
Да, в этом коде `time`, `money` и `number` являются модификаторами элемента `item` блока `info-box`.

**Comment by Roman52** [comment]
Спасибо!

**Comment by AlexanderBukhtaty** [comment]
Здравствуйте аналогичный вопрос но только на следующем примере
Делаю блок Chart (диаграмма), у него есть модификатор легенда(булевый - рендерить или нет легенду)
саму лугенду реализовал как элемент блока chart:
``` html
<div class="chart chart_type_donut chart_legend i-bem chart_js_inited">
<div class="chart__graphic">
<svg>...</svg>
</div>
<div class="chart__legend">
    <div class="chart__legend-item ">
        <div class="chart__legend-badge chart__legend-badge_inline"></div>
        <div class="chart__legend-value chart__legend-value_inline">60</div>
        <div class="chart__legend-text chart__legend-text_inline">Мужской</div>
    </div>
    <div class="chart__legend-item ">
        <div class="chart__legend-badge chart__legend-badge_inline"></div>
        <div class="chart__legend-value chart__legend-value_inline">40</div>
        <div class="chart__legend-text chart__legend-text_inline">Женский</div>
    </div>
</div>
</div>
```
у меня долго болела голова правильно ли я сделал!? указал все элементы легенты как элементы блока диаграммы (chart)
просто по логике badge,value,text являются элементами легенды, а та в свою очередъ элементом chart
а элементы к элементам не пишутся.
легенду как самостоятельный болк нет смысла реализовывать так как она только к графикам и применима?
=) вот такая каша у меня в голове.

**Comment by tadatuta** [comment]
@AlexanderBukhtaty да, в приведенном коде все хорошо

**Comment by AlexanderBukhtaty** [comment]
@tadatuta Спасибо
